src/Raven.Server/Config/RavenConfiguration.cs

```diff
@@ -135,6 +137,15 @@ private void AddJsonConfigurationVariables(string customConfigPath = null)
             }
         }
 
+        private void CeateConfigFileFromDefaultIfNeeded()
+        {
+            if (File.Exists("settings.json") == false &&
+                File.Exists("settings.default.json")) //just in case
+            {                
+                File.Move("settings.default.json","settings.json");
```

Comment:
Can we do copy instead? If somebody wants to start clean he would just delete his current `settings.json`.

Reply:
Also the code does not consider our custom config path switch - `-c`.
https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Server/Utils/Cli/CommandLineSwitches.cs#L24

Then if `customConfigPath` switch value is not empty, we should be checking and copying there instead of the default - `settings.json` like so:
```csharp
var targetSettingsFile = string.IsNullOrEmpty(CommandLineSwitch.CustomConfigPath)
    ? "settings.json"
    : customConfigPath
if (File.Exists(targetSettingsFile ) == false &&
    File.Exists("settings.default.json")) //just in case
{                
     File.Move("settings.default.json",targetSettingsFile );
```

We should move this to `Program.cs` and run right after we load command line switches instead of `RavenConfiguration`.